core/filters/test/unit/price.spec.ts

```diff
@@ -0,0 +1,64 @@
+import { price } from '@vue-storefront/core/filters/price'
+
+jest.mock('@vue-storefront/core/lib/multistore', () => ({
+  currentStoreView: jest.fn().mockReturnValue({
+    i18n: {
+      defaultLocale: 'en-US',
+      currencySign: '$',
+      currencyDecimal: '',
+      currencyGroup: '',
+      fractionDigits: 2,
+      priceFormat: '{sign}{amount}'
+    }
+  })
+}))
+
+describe('price', () => {
+  it('Check if argument is not a number', () => {
+    expect(price(undefined)).toBe(undefined)
+  })
+
+  it('Check if number is converted to string with sign', () => {
+    expect(price('20.99')).toBe('$20.99')
+  })
+
+  it('Check if i18n doesn\'t exist', () => {
```

Comment:
Rather `returns input if 'i18n' is not present in storeView object`. You are checking here, what happens if `i18n doesn't exist` not if it exists

Reply:
Do not copy-paste, make sure you understand it and keep this knowledge for the future